paths/orders/create.yaml

```diff
@@ -48,6 +48,9 @@ requestBody:
       schema:
         type: object
         title: order/create/parameters
+        required:
+          - name
+          - lsp
```

Comment:
`source_locale_id` should also be considered as mandatory? Considering its validated in the model?

Reply:
Just checked and if it's empty the project's default locale will be used